Automattic/jetpack
# VideoPress: Introduce `VideoPressUploader` component

This is the first step to improving the uploading experience for VP new block.

Based on the idea of splitting the VideoPress experience into 2 moments: `Uploading` and `Previewing`, having a specific component for handling the `Upload` part could make easier the time to implement new uploading features such as uploading editor ( #24556 ) or error improvements.

#### Changes proposed in this Pull Request:

* Extract upload logic to `VideoPressUploader`.
* Keep the current behavior.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Jetpack` plugin with support of `beta blocks`.
* Interact with `VideoPress (beta)`.
* Nothing should change in the experience.
